assets/js/utils/rtcUtils.ts

```diff
@@ -140,13 +159,23 @@ export const connectWebrtc = async (
       onJoinError: () => {
         onError("Error while joining WebRTC connection");
       },
-      onTrackReady: ({ track, peer }) => {
-        if (track != null)
-          addOrReplaceTrack(peer.metadata, track, playerCallbacks[peer.metadata.email]);
-      },
       onTrackAdded: () => {
         // do nothing
       },
+      onTrackReady: ({ track, peer, metadata }: TrackContext) => {
+        if (track != null) {
+          addOrReplaceTrack(peer.metadata, track, playerCallbacks[peer.metadata.email]);
+          if (metadata.enabled) track.enabled = metadata.enabled;
+          updateMicAndCamStatusCallback();
+        }
+      },
+      onTrackUpdated({ track, metadata }: TrackContext) {
+        if (track != null) {
+          if (metadata.enabled) track.enabled = metadata.enabled;
```

Comment:
Why can't we always set `track.enabled` to `metadata.enabled`?

Reply:
Yeah, that's not what I've meant... it should clearly be `'enabled' in metadata`. Changed it